src/plopp/backends/backend_manager.py

```diff
@@ -0,0 +1,77 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022 Scipp contributors (https://github.com/scipp)
+
+
+class BackendManager(dict):
```

Comment:
Putting code for all backends into a single class might not scale very well. Instead, you could have a separate class per backend of the form
```python
class MatplotlibBackend:
    @property
    def Canvas2d(self):
        from .matplotlib import Canvas as CanvasMpl
        return CanvasMpl

    ...
```
and `BackendManager` could store these backend classes and access them via
```python
class BackendManager(dict):
   @property
    def Canvas2d(self):
        try:
            return self.backends[self['2d']].Canvas2d
        except KeyError:
            raise ValueError(f'Unsupported backend \'{self["2d"]}\' for Canvas2d.')

    ...
```